chapters/Chapter 2 — Dressara.md

```diff
@@ -0,0 +1,94 @@
+
+There only a few thousand inhabitants on Dressara, plus their few thousand servants. There was a party nearly every day, and everyone was invited—sometimes the servants, too. The men, Leanne knew, did little more than stand about all day, whether at parties or at home. They were utterly boring, both as people and as targets. The women, Leanne, assumed, did something more than simply wear dresses and stroll about—even the wealthy had to bring in money somehow—but whatever they did, it wasn’t interesting enough for Leanne to learn what it was. 
+
+But if any Dressaran woman—or, for that matter, any Dressaran at all—were ever to do anything interesting, Leanne would bet it would be Delphine. 
+
+Leanne looked away as Delphine approached. Her neck muscles tensed, her back straightened, her eyes glanced down—
+
+She jumped as a hand tapped her bare shoulder.
+
+“My apologies,” said a deep, melodic voice. “I didn’t mean to startle you.”
+
+It was Delphine. Leanne tore her eyes from her lips—halfway between smile and smirk, just as Leanne had seen before, and there was no mistaking that dark lipstick—up to Delphine’s eyes, kind and stalwart, framed by her short black hair.
+
+“A red,” Delphine instructed. “Merlot.”
+
+Leanne nodded readily. Her heat beat unevenly as she felt herself move towards the back of the garden, where the bar was.
+
+She had considered inviting herself to the garden party. She’d done as much before: the security at Dressaran parties was nonexistent: Dressarans tended to assume they were safe at all times. If they didn’t recognize her, they’d assume their Alt was forgetful. They’d make smalltalk just fine, and be friendly enough.
+
+But while Dressarans rarely had much of substance to say, when they did, they generally only shared it with those they knew. Even those that recognized Leanne from her previous intrusions would not feel so comfortable around her.
+
+Servants, however, were generally invisible. Their clothing depended on the party; today, they wore shimmering gray tunics held up by silvery strings that snaked over their shoulders.  
+
+Leanne shivered as she poured the wine. She was unused to exposing so much skin to the elements. The artificial wind tickled the scar upon her shoulder, the scar Delphine had tapped, the scar Leanne had received on the moon Delphinia after thirty seven assassins had cornered her. 
+
+She assumed Delphine was named after Delphinia. As much as the Dressarans looked down upon the other planets and moons, they frequently named their children after them: “Delphine,” “Delphy,” “Elise,” “Elsa.”
+
+Elsa was Delphine, Leanne assumed. She looked different now, of course. Her face had been soft, now it was angular. Her hair long and red, now short and black. But there was no mistaking her.
```

Comment:
Although Delphine is not trans, referring to Delphine's past identity as if it is distinct from Delphine makes me uncomfortable. 

I can't think of a good way around this.

> Delphine was her target, Leanne assumed. She looked different now, of course. On the Leesomes' card, her name hadn't been Delphine. Her face had been soft, now it was angular; her hair long and red, now short and black.  But there was no mistaking her.